### PR TITLE
add ECON1101

### PR DESCRIPTION
**Pull request by TLNA114514**
Add a new course

### CODE REVIEW

**Comment by why-in-Shanghaitech** [comment]
Hi! Thank you very much for your contribution! I'll review this PR in a few days.
I've noticed that this is your first contribution. I'll add you to the contributor list once the pull request is accepted.

File: courses/ECON1101/meta.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "code": "ECON1101",
```
Comment (why-in-Shanghaitech) [inline courses/ECON1101/meta.json:2]: From the slides, I've noticed that the course code is ECON1001. Which one is correct?

File: courses/ECON1101/meta.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        {
+            "season": "Fall",
+            "year": "2023",
+            "course_id": "ECON1101.01",
+            "department": "信息科学与技术学院",
+            "credit": "3",
+            "hours": "48",
+            "prerequisite": "无",
+            "evaluation": "20% homework + 15% quiz + 15% midterm + 50% final",
+            "teacher": [],
```
Comment (why-in-Shanghaitech) [inline courses/ECON1101/meta.json:14]: I've noticed that the instructor is Xiyi Yang and the TA is Dong Zhang. Is this correct? If you could confirm this info, I could fix the `teacher` and `ta` lists for you.

**Review by why-in-Shanghaitech — COMMENTED** [review]
Waiting for author response...

File: courses/ECON1101/meta.json (hunk 1 above)
Comment (why-in-Shanghaitech) [reply courses/ECON1101/meta.json:2]: Confirmed from [LRC](https://lrc.shanghaitech.edu.cn/course-detail?course_id=1323082), the course code ECON1101 is correct.

**Comment by TLNA114514** [comment]
Sorry for replying so late, for I didn't frequently check my outlook.
 The correct course code is ECON1101. I typed the wrong number at the beginning. I'm sorry for the mistake I had taken and the inconvenience I might bring to you.
发自 Outlook<http://aka.ms/weboutlook>
________________________________
发件人: Haoyi Wu ***@***.***>
发送时间: 2024年1月29日 19:14
收件人: i-TechX/iTechX ***@***.***>
抄送: TLNA114514 ***@***.***>; Author ***@***.***>
主题: Re: [i-TechX/iTechX] add ECON1101 (PR #106)


@why-in-Shanghaitech commented on this pull request.

________________________________

In courses/ECON1101/meta.json<https://github.com/i-TechX/iTechX/pull/106#discussion_r1469440180>:

> @@ -0,0 +1,18 @@
+{
+    "code": "ECON1101",


Confirmed from LRC<https://lrc.shanghaitech.edu.cn/course-detail?course_id=1323082>, the course code ECON1101 is correct.

―

**Comment by TLNA114514** [comment]
The names of the instructor and TA are right.  Thank you for your correction！

发自 Outlook<http://aka.ms/weboutlook>
________________________________
发件人: Haoyi Wu ***@***.***>
发送时间: 2024年1月29日 19:10
收件人: i-TechX/iTechX ***@***.***>
抄送: TLNA114514 ***@***.***>; Author ***@***.***>
主题: Re: [i-TechX/iTechX] add ECON1101 (PR #106)


@why-in-Shanghaitech commented on this pull request.

Waiting for author response...

________________________________

In courses/ECON1101/meta.json<https://github.com/i-TechX/iTechX/pull/106#discussion_r1469432620>:

> @@ -0,0 +1,18 @@
+{
+    "code": "ECON1101",


From the slides, I've noticed that the course code is ECON1001. Which one is correct?

________________________________

In courses/ECON1101/meta.json<https://github.com/i-TechX/iTechX/pull/106#discussion_r1469434829>:

> @@ -0,0 +1,18 @@
+{
+    "code": "ECON1101",
+    "name": "Introduction to Economics: Energy and Environmental Economics\t",
+    "semesters": [
+        {
+            "season": "Fall",
+            "year": "2023",
+            "course_id": "ECON1101.01",
+            "department": "信息科学与技术学院",
+            "credit": "3",
+            "hours": "48",
+            "prerequisite": "无",
+            "evaluation": "20% homework + 15% quiz + 15% midterm + 50% final",
+            "teacher": [],


I've noticed that the instructor is Xiyi Yang and the TA is Dong Zhang. Is this correct? If you could confirm this info, I could fix the teacher and ta lists for you.

―

**Review by why-in-Shanghaitech — APPROVED** [review]
Thank you so much for your quick reply. I'll merge this PR after fixing the `teacher` and `ta` lists.

**Pull request merged**